test/test_prototype_transforms_functional.py

```diff
@@ -175,7 +175,7 @@ def test_cuda_vs_cpu(self, info, args_kwargs):
 
     @sample_inputs
     @pytest.mark.parametrize("device", cpu_and_gpu())
-    def test_dtype_and_device_consistency(self, info, args_kwargs, device):
+    def test_dtype_consistency(self, info, args_kwargs, device):
```

Comment:
I've opted to split dtype and device consistency tests. Not sure if this is the right way to go. If we don't, we can only skip this completely for the dtype conversion kernels.